Jenkinsfile

```diff
@@ -35,7 +33,11 @@ void buildManifest(String variant_name, String bitbake_image) {
 
         // From here on, we run these commands inside the docker container. The
         // default pwd inside is the path to the Jenkins workspace.
+        String uid = sh(returnStdout: true, script: "id -u").trim()
```

Comment:
This is never used.

Reply:
I know, this whole thing is WIP for now.